Blockzilla/en.lproj/Localizable.strings

```diff
@@ -343,6 +343,9 @@
 /* %@ is the name of the app (Focus / Klar). Description for 'Enable Face ID' displayed under its respective toggle in the settings menu. */
 "Settings.toggleFaceIDDescription" = "Face ID can unlock %@ if a URL is already open in the app";
 
+/* Show home screen tips toggle label on settings screen */
```

Comment:
This should be inside UIConstants.strings

Reply:
Can you clarify what you mean? I added this to `UIConstants.strings` as well: https://github.com/mozilla-mobile/focus-ios/blob/9b23ca04272e2edec4bb984dc9ef744bb2009018/Blockzilla/UIConstants.swift#L301

Reply:
Since it's in UIConstants, you don't need to add it to the Localizable.strings file (we have a script that adds it to all our localizations). So you can just delete this change if you made it manually.

Reply:
Ah got it, thanks!